### PR TITLE
Changed FocusPicker, TagPicker, and DatePicker to buttons

### PR DESCRIPTION
**Pull request by IshikaJain**
### Summary 
Changed span tags for all the pickers to button tags. Adjusted the corresponding css to maintain the same design by adding more css classes for each picker. Each picker still performs the same as before, but internally each picker is a button now. 

### Fixes
Mostly changed border-color, color, background-color, and padding to get the focus picker to look like before it was changed to a button - made a new class with these attributes
Changed border-color, color, font, width and height to get the tag and date picker to look like before it was changed to a button- made a new class called "TagButton" and DateButton  to integrate these changes

- [x] changed FocusPicker to button
- [x] changed DatePicker to button
- [x] changed TagPicker to button
- [x] fixes https://github.com/cornell-dti/samwise/issues/491#issue-612272465

### Test Plan 

Compared the original functionality from before making the changes to after making each picker a button. Original functionality/appearance: 
Screen Shot 2020-10-02 at 11 28 38 PM
Screen Shot 2020-10-02 at 11 28 32 PM
Screen Shot 2020-10-02 at 11 28 22 PM

Functionality/appearance after making changes:
Screen Shot 2020-10-02 at 11 30 58 PM
Screen Shot 2020-10-02 at 11 30 53 PM
Screen Shot 2020-10-02 at 11 30 46 PM

I compared the appearance/functionality after making the changes to the original version to make sure the designs stayed the same and the pickers performed as expected

### CODE REVIEW

**Comment by SamChou19815** [comment]
@IshikaJain linter and tests are failing. Please fix all linter errors and follow the instruction of `yarn test` to update snapshots.

File: frontend/src/components/TaskCreator/DatePicker.tsx [hunk 1]
```diff
@@ -132,15 +132,16 @@ export default function DatePicker(props: Props): ReactElement {
       </>
     );
     return (
-      <span
-        role="presentation"
+      <button
+        // role="presentation"
```
Comment (ptwu) [inline frontend/src/components/TaskCreator/DatePicker.tsx:136]: Should remove commented code before merging to master, especially if it should be deleted

File: frontend/src/components/TaskCreator/TagPicker.tsx [hunk 2]
```diff
@@ -46,15 +46,17 @@ function TagPicker({ tag, opened, onTagChange, onPickerOpened, getTag }: Props):
       </>
     );
     return (
-      <span
-        role="presentation"
+      <button
+        type="button"
+        // role="presentation"
```
Comment (ptwu) [inline frontend/src/components/TaskCreator/TagPicker.tsx:51]: Same here

File: frontend/src/components/TaskCreator/Picker.module.scss [hunk 3]
```diff
@@ -17,6 +17,35 @@
...
+  width: 96.84px !important;
+  height: 20px !important;
+  padding: 0px !important;
+}
+
+.DateButton {
+  background: rgb(150, 150, 150) !important;
+  border-color: transparent !important;
+  color: #ffffff !important;
+  // padding: 0px 2px 0px 10px !important;
```
Comment (ptwu) [inline frontend/src/components/TaskCreator/Picker.module.scss:42]: Same here

File: frontend/src/components/TaskCreator/Picker.module.scss [hunk 4]
```diff
@@ -17,6 +17,35 @@
...
+  color: transparent !important;
+  background-color: transparent !important;
+  padding: 0px;
+}
+
+.TagButton {
+  background: rgb(150, 150, 150) !important;
+  border-color: transparent !important;
+  color: #ffffff !important;
+  // padding: 0px 2px 0px 10px !important;
```
Comment (ptwu) [inline frontend/src/components/TaskCreator/Picker.module.scss:31]: More commented code

File: frontend/src/components/TaskCreator/Picker.module.scss [hunk 5]
```diff
@@ -17,6 +17,35 @@
...
+  padding: 0px !important;
+}
+
+.DateButton {
+  background: rgb(150, 150, 150) !important;
+  border-color: transparent !important;
+  color: #ffffff !important;
+  // padding: 0px 2px 0px 10px !important;
+  font: 14px 'Open Sans', sans-serif !important;
+  width: 94.03px !important;
```
Comment (ptwu) [inline frontend/src/components/TaskCreator/Picker.module.scss:44]: Are all of these `!important`s necessary?

File: frontend/src/components/TaskCreator/Picker.module.scss [hunk 6]
```diff
@@ -29,14 +58,16 @@
 .DateDisplay {
   display: inline-flex;
   align-items: center;
-  padding: 0 5px 0 10px;
+  //padding: 0 5px 0 10px;
```
Comment (pihucode) [inline frontend/src/components/TaskCreator/Picker.module.scss:61]: You can remove this commented line here.

**Review by jason-tung — APPROVED** [review]

**Pull request merged**